doc/devel/contributing.rst

```diff
@@ -102,34 +112,39 @@ A brief overview is:
 
 3. Clone this copy to your local disk::
 
-      $ git clone https://github.com/YourLogin/matplotlib.git
+      git clone https://github.com/<YOUR GITHUB USERNAME>/matplotlib.git
+
+4. Enter the directory and install the local version of Matplotlib. For more
+   detailed instructions on doing this see ref`<installing_for_devs>`::
 
-4. Create a branch to hold your changes::
+      cd matplotlib
+      pip install -ve . 
```

Comment:
This is only half the truth. It's important to set up a dedicated environment (venv or conda) for development and do the editable install there. Otherwise people will corrupt their standard python environment.

I suggest not giving any details here but completely deferring to ``:ref:`installing_for_devs` ``.